plugins/cli/cli.go

```diff
@@ -53,6 +42,6 @@ func printUsage() {
 	)
 	flag.PrintDefaults()
 
-	fmt.Fprintf(os.Stderr, "\nThe following plugins are enabled by default and can be disabled with -%s:\n  %s\n", getFlagName(node.DISABLE_PLUGINS.Name), getList(enabledPlugins))
-	fmt.Fprintf(os.Stderr, "The following plugins are disabled by default and can be enabled with -%s:\n  %s\n\n", getFlagName(node.ENABLE_PLUGINS.Name), getList(disabledPlugins))
+	fmt.Fprintf(os.Stderr, "\nThe following plugins are enabled: %s\n", getList(parameter.NodeConfig.GetStringSlice(node.CFG_ENABLE_PLGUINS)))
```

Comment:
When a plugin is created in the code it takes a parameter whether it is enabled by default. The idea of these two lines in the previous version was to loop through all the plugins registered in GoShimmer and display them here and say whether they are enabled/disabled (to get this information, they are added to `enabledPlugins` `disabledPlugins`). That way you could see all the available plugins and what actually makes sense to enable/disable as a parameter.

I guess it makes sense to have that behavior again. An alternative would be to remove the enabled-flag from the plugin itself completely and only use CFG with default values for that. However, that way, when a new plugin is written that is not added to the defaults, the user couldn't see it and its name in the usage screen.

Reply:
I've reverted it back to the two old print outs (just with the adjusting flag). Note that because the config file can also hold `node.enablePlugins/node.disablePlugins` but is only loaded after `flag.Parse()` is called, its defined enabled/disabled plugins are not part of the usage output.

Reply:
I guess that is acceptable!